hack/kubevirt-builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM kubevirt/builder@sha256:112c6515515ece0949ea566cc911d884df9be1224980825d3d0ea6d3072ace0c
+FROM murilofv/builder:30-5.4.0
```

Comment:
Where can I find this image?
Probably we should build the builder image for PPC as well and push it under the kubevirt docker repo

Reply:
I put it in [dockerhub](https://hub.docker.com/r/murilofv/builder/tags) , the 30-5.4.0 tag is a manifest that points to the 30-5.4.0 builder in the kubevirt dockerhub for amd64 and for ppc64le to the 30-5.4.0-ppc64le in the same link, that way this one pointer can be used by both architectures.

Reply:
Can you add the dockerfile under the https://github.com/kubevirt/kubevirt/tree/master/hack/builder
We already build libvirt container for both x86_64 and ppc64le, it can be perfect if you can adapt our builder scripts similar to https://github.com/kubevirt/libvirt/blob/master/Dockerfile

Reply:
The Dockerfile is exactly the same as it is, I didn't have to make it any changes to build it for Power.

Not sure if I fully understood your second request, as far as I understand the kubevirt builder is being manually pushed to the dockerhub, as opposed to the libvirt one that is leveraging the Travis CI in order to build the containers in all archs and is being responsible to push them (along with the manifest). So are you thinking of heading the Travis CI path or want to keep the builds manual and have some additional functionalities in the scripts like tagging images built with their arch / building the manifest? Keep in mind that the second option would require access to a Power system in order to build the builder image, which I suggested minicloud in a previous comment to see if it would attend the need.

Reply:
I think we are good this way. Just need to sort out the building of that image from our side ...